app/main.py

```diff
@@ -16,4 +16,17 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+variables_list = [lucky_number, pi,
+                  one_is_a_prime_number, name,
+                  my_favourite_films, profile_info,
+                  marks, collection_of_coins]
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+for element in variables_list:
```

Comment:
The solution is ok, but it will be much less code if you sort variables in the dict manually. Try that

Reply:
I thought I must write a code, which will do this automatically, but thanks for clarifying)